config.json

```diff
@@ -13,7 +13,7 @@
   bind: {
     // Listen on a network socket
     socket: {
-      address: "127.0.0.1",
+      address: "0.0.0.0",
```

Comment:
I need to look more into why this is necessary. I think Compose might be doing some proxying to make ports appear on the host and then forward into the network it creates.